frontend/nginx-conf/diamonds.conf

```diff
@@ -1,12 +1,20 @@
 server {
     listen 80;
-    server_name localhost;
+    server_name diamonds.etimo.se;
 
     error_page 401 403 404 /404.html;
 
     root /usr/share/nginx/html;
     index index.html;
 
+    location /.well-known/acme-challenge/ {
+    root /var/www/certbot;
+    }
+
+    location / {
+        return 301 https://diamonds.etimo.se$request_uri;
```

Comment:
Can we redirect to ourself again? Wont that cause infinite redirect loop?

Reply:
This is supposed to redirect requests to HTTPS instead of HTTP.

Reply:
Ah yes I see now, missed that